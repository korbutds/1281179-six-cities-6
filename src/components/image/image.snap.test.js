import React from 'react';
import {render} from '@testing-library/react';
import Image from './image';

test(`Shoul render Image component correctly`, () => {
  const image = `fakeurl`;
  const {container} = render(<Image image={image} />);
  expect(container).toMatchSnapshot();
});
