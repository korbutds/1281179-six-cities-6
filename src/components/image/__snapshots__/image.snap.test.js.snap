// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shoul render Image component correctly 1`] = `
<div>
  <div
    class="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      class="property__image"
      src="fakeurl"
    />
  </div>
</div>
`;
